### DISCUSSION TITLE
Have two cars in WeConnect

### DISCUSSION THREAD

**Discussion by kennls** [tillsteinbach/VWsFriend#98] [Q&A] [answered]
I have a iD.3 and a VW California. The California "steals" the HomeKit lock.

Any way you can ignore a car by stating the VW ID number, of the car you want to ignore?

**Comment by tillsteinbach** [comment]
What do you mean with "steals"? The ID.3 cannot provide the lock status at the moment.

**Comment by kennls** [comment]
Here are two screenshots:



ons. 10. nov. 2021 kl. 19.48 skrev Till Steinbach ***@***.***
>:

> What do you mean with "steals"? The ID.3 cannot provide the lock status at
> the moment.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tillsteinbach/VWsFriend/discussions/98#discussioncomment-1621733>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWNWT72ELCDNERIJ3S4BAALULK5A3ANCNFSM5HYPRSLA>
> .
>
-- 
Venlig Hilsen/Kind regards
______________________

Kenn L. Schjødt
***@***.***
Mobile: +45 3059 6597
Mobile: +45 9397 0397
______________________

**Reply by tillsteinbach** [reply]
I guess you replied to the GitHub mail. Unfortunately the screenshots are missing here

**Reply by kennls** [reply]
![image](https://user-images.githubusercontent.com/94071295/141177860-11bd25e5-50e5-45a5-9f6f-2ec765fd3696.png)

**Reply by kennls** [reply]
![image](https://user-images.githubusercontent.com/94071295/141177923-ce7e565c-def8-43c2-85fa-cb19e92d2027.jpeg)

**Reply by kennls** [reply]
![image](https://user-images.githubusercontent.com/94071295/141178783-7f6079e5-f315-41e4-96e7-e7fa9b7de883.jpeg)

**Reply by kennls** [reply]
![image](https://user-images.githubusercontent.com/94071295/141178826-f88bdeda-349d-4c14-bb4b-01341aff4f53.jpeg)

**Reply by tillsteinbach** [reply]
That is interesting! It looks like you are the first one with a pure fossil car, so you will experience some problems :) but we can try to fix those piece by piece.
The HomeKit devices actually look good. The ID.3 does not provide the lock status with its current firmware, but it looks like your California does. So this is probably fine.
Regarding the missing status picture, maybe VW does not have a picture on the server for your car. I would need to have more insight to debug that.

**Reply by kennls** [reply]
I can send the log files tomorrow in a private message :)

**Reply by tillsteinbach** [reply]
Sounds good. And we should try to make you able to login to Grafana. Will be interesting if there are further problems with the California as it does not have a battery.

**Reply by kennls** [reply]
![image](https://user-images.githubusercontent.com/94071295/141277614-5f0ad699-14fc-4ad7-9f0f-c454a117e2be.jpeg)

**Reply by kennls** [reply]
schjodtubuntu@ubuntu-server:~$ docker exec c2324986b229 bash -c "echo @GF_SECURITY_ADMIN_USER"
@GF_SECURITY_ADMIN_USER
schjodtubuntu@ubuntu-server:~$

**Reply by tillsteinbach** [reply]
I think the way to check the variable should look something like:
`docker exec vwsfriend_grafana_1 bash -c 'echo  "$GF_SECURITY_ADMIN_PASSWORD"'`

**Reply by kennls** [reply]
The VW California lock works, in that it displays the lock status. But it is just as slow, as the WeConnect app, to update the lock status. But that isnt your fault, just think the car/server traffic is very slow.

Tried your docker suggestion, but it didn't work. It only works by stating the container id. Here is my containers:

CONTAINER ID   IMAGE                                    COMMAND                  CREATED        STATUS                       PORTS                                       NAMES
0cb5ec7accce   tillsteinbach/vwsfriend:latest           "/bin/sh -c 'vwsfrie…"   17 hours ago   Up About an hour (healthy)                                               vwsfriend-main_vwsfriend_1
8b881956fa8d   postgres:13                              "docker-entrypoint.s…"   17 hours ago   Up About an hour (healthy)   0.0.0.0:5432->5432/tcp, :::5432->5432/tcp   vwsfriend-main_postgresdb_1
c2324986b229   tillsteinbach/vwsfriend-grafana:latest   "/run.sh"                17 hours ago   Up About an hour (healthy)   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp   vwsfriend-main_grafana_1

Output running the different commands.

schjodtubuntu@ubuntu-server:~$ docker exec vwsfriend_grafana_1 bash -c 'echo "@GF_SECURITY_ADMIN_USER"'
Error: No such container: vwsfriend_grafana_1
schjodtubuntu@ubuntu-server:~$ docker exec tillsteinbach/vwsfriend_grafana_1 bash -c 'echo "@GF_SECURITY_ADMIN_USER"'
Error: No such container: tillsteinbach/vwsfriend_grafana_1
schjodtubuntu@ubuntu-server:~$ docker exec c2324986b229 bash -c 'echo "@GF_SECURITY_ADMIN_USER"'
@GF_SECURITY_ADMIN_USER

**Reply by tillsteinbach** [reply]
ah yes, the container name is different for you, it is not `vwsfriend_grafana_1` in your case but `vwsfriend-main_grafana_1`

`docker exec vwsfriend-main_grafana_1 bash -c 'echo "$GF_SECURITY_ADMIN_PASSWORD"'` would be your command

**Reply by kennls** [reply]
And secret is the password

schjodtubuntu@ubuntu-server:~$ docker exec vwsfriend-main_grafana_1 bash -c 'echo "$GF_SECURITY_ADMIN_PASSWORD"'
secret

So will check the user

**Comment by kennls** [comment, marked as answer]
And the user is not admin, its the computername:

schjodtubuntu@ubuntu-server:~$ docker exec vwsfriend-main_grafana_1 bash -c 'echo "$GF_SECURITY_ADMIN_USER"'
schjodtubuntu

**Reply by tillsteinbach** [reply]
Ok, but where does that come from. Really strange! Will have to investigate

**Reply by tillsteinbach** [reply]
I think I found out. On some systems you cannot set USERNAME variable. It always resolves to your user. I will rename it in the yml and env template.

**Reply by tillsteinbach** [reply]
Everything should be fixed with the changes I made to the docker-compose and .env files now. Thanks for finding this out with me!

**Reply by kennls** [reply]
I am the thankful...its a great app and happy to help :)

**Comment by kennls** [comment]
And it works now :)
